### PR TITLE
Zoom controls

### PR DESCRIPTION
**Pull request by thisandagain**
### Resolves

GH-43

### Proposed Changes

- Makes trashcan hidden by default
- Updates zoom controls as per #43 and discussion with @carljbowman 
- Replaces use of `sprites.png` with individual SVG assets for each zoom control (easier to maintain)

### CODE REVIEW

**Comment by thisandagain** [comment]
Note to self: squash during merge

File: core/zoom_controls.js [hunk 1]
```diff
@@ -40,32 +40,60 @@ Blockly.ZoomControls = function(workspace) {
...
+ * @private
+ */
+Blockly.ZoomControls.prototype.ZOOM_OUT_PATH = 'zoom-out.svg';
+
+/**
+ * Zoom reset icon path.
+ * @type {string}
+ * @private
+ */
+Blockly.ZoomControls.prototype.ZOOM_RESET_PATH = 'zoom-reset.svg';
```
Comment (rschamp) [inline core/zoom_controls.js]: Nit on these, private vars conventionally end in `_`

**Review by rschamp — CHANGES_REQUESTED** [review]
Just one syntax thing — tested it out in the GUI and the VM, looks/works great!

**Comment by thisandagain** [comment]
@rschamp Good catch. Fixed.

**Review by rschamp — APPROVED** [review]

**Pull request merged**